<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessagelistControl</class>
 <widget class="QWidget" name="MessagelistControl">
   <property name="AccountType">
     <number>1</number>
   </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>295</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="inboxLayout">
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item>
    <widget class="SSplitter" name="horizontalSplitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="inboxIdentitiesWidget">
      <layout class="QVBoxLayout">
       <item>
        <widget class="TreeWidgetIdentities" name="treeWidget">
         <property name="baseSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Identities</string>
          </property>
          <property name="icon">
           <iconset>
            <selectedoff>:/newPrefix/images/identities.png</selectedoff>
           </iconset>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonNew">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string notr="true">New</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="SSplitter" name="verticalSplitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="searchWidget">
       <layout class="QHBoxLayout" name="searchLayout">
        <item>
         <widget class="QLineEdit" name="searchLineEdit">
          <property name="placeholderText">
           <string>Search</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="searchOption">
          <item>
           <property name="text">
            <string>All</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>To</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>From</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Subject</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Message</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="InboxMessagelist" name="messagelistInbox">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>200</number>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>27</number>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>26</number>
       </attribute>
      </widget>
      <widget class="MessageView" name="messageView">
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>500</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="plainText">
        <string>
        Welcome to easy and secure Bitmessage
            * send messages to other people
            * send broadcast messages like twitter or
            * discuss in chan(nel)s with other people
	      </string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MessageView</class>
   <extends>QTextEdit</extends>
   <header>bitmessageqt.messageview</header>
  </customwidget>
  <customwidget>
   <class>STreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>bitmessageqt.settingsmixin</header>
  </customwidget>
  <customwidget>
   <class>SSplitter</class>
   <extends>QSplitter</extends>
   <header>bitmessageqt.settingsmixin</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>InboxMessagelist</class>
   <extends>QTableView</extends>
   <header>bitmessageqt.messagelist</header>
  </customwidget>
  <customwidget>
   <class>TreeWidgetIdentities</class>
   <extends>QTreeWidget</extends>
   <header>bitmessageqt.messagelist</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="bitmessage_icons.qrc"/>
 </resources>
 <connections>
  <!--  <connection> -->
  <!--  <sender>messagelistInbox</sender> -->
  <!--  <signal>customContextMenuRequested(QPoint)</signal> -->
  <!--  <receiver>MainWindow</receiver> -->
  <!--  <slot>on_context_menuInbox()</slot> -->
  <!--  <hints> -->
  <!--   <hint type="sourcelabel"> -->
  <!--    <x>656</x> -->
  <!--    <y>366</y> -->
  <!--   </hint> -->
  <!--   <hint type="destinationlabel"> -->
  <!--    <x>442</x> -->
  <!--    <y>289</y> -->
  <!--   </hint> -->
  <!--  </hints> -->
  <!-- </connection> -->
  <!-- <connection> -->
  <!--  <sender>pushButtonNewAddress</sender> -->
  <!--  <signal>clicked()</signal> -->
  <!--  <receiver>MainWindow</receiver> -->
  <!--  <slot>click_NewAddressDialog</slot> -->
  <!--  <hints> -->
  <!--   <hint type="sourcelabel"> -->
  <!--    <x>20</x> -->
  <!--    <y>20</y> -->
  <!--   </hint> -->
  <!--   <hint type="destinationlabel"> -->
  <!--    <x>20</x> -->
  <!--    <y>20</y> -->
  <!--   </hint> -->
  <!--  </hints> -->
  <!-- </connection> -->
  <!-- <connection> -->
  <!--  <sender>treeWidgetYourIdentities</sender> -->
  <!--  <signal>customContextMenuRequested(QPoint)</signal> -->
  <!--  <receiver>MainWindow</receiver> -->
  <!--  <slot>on_context_menuYourIdentities()</slot> -->
  <!--  <hints> -->
  <!--   <hint type="sourcelabel"> -->
  <!--    <x>20</x> -->
  <!--    <y>20</y> -->
  <!--   </hint> -->
  <!--   <hint type="destinationlabel"> -->
  <!--    <x>20</x> -->
  <!--    <y>20</y> -->
  <!--   </hint> -->
  <!--  </hints> -->
  <!-- </connection> -->
  <!-- <connection> -->
  <!--  <sender>inboxSearchLineEdit</sender> -->
  <!--  <signal>returnPressed()</signal> -->
  <!--  <receiver>MainWindow</receiver> -->
  <!--  <slot>inboxSearchLineEditReturnPressed</slot> -->
  <!--  <hints> -->
  <!--   <hint type="sourcelabel"> -->
  <!--    <x>20</x> -->
  <!--    <y>20</y> -->
  <!--   </hint> -->
  <!--   <hint type="destinationlabel"> -->
  <!--    <x>20</x> -->
  <!--    <y>20</y> -->
  <!--   </hint> -->
  <!--  </hints> -->
  <!-- </connection> -->
  <!-- <connection> -->
  <!--  <sender>inboxSearchLineEdit</sender> -->
  <!--  <signal>textChanged(QString)</signal> -->
  <!--  <receiver>MainWindow</receiver> -->
  <!--  <slot>inboxSearchLineEditUpdated</slot> -->
  <!--  <hints> -->
  <!--   <hint type="sourcelabel"> -->
  <!--    <x>20</x> -->
  <!--    <y>20</y> -->
  <!--   </hint> -->
  <!--   <hint type="destinationlabel"> -->
  <!--    <x>20</x> -->
  <!--    <y>20</y> -->
  <!--   </hint> -->
  <!--  </hints> -->
  <!-- </connection> -->
  <connection>
   <sender>treeWidget</sender>
   <signal>currentItemChanged(QTreeWidgetItem*, QTreeWidgetItem*)</signal>
   <receiver>messagelistInbox</receiver>
   <slot>folderChanged</slot>
  </connection>
  <!-- <connection> -->
  <!--  <sender>treeWidgetYourIdentities</sender> -->
  <!--  <signal>itemChanged(QTreeWidgetItem*,int)</signal> -->
  <!--  <receiver>MainWindow</receiver> -->
  <!--  <slot>treeWidgetItemChanged</slot> -->
  <!--  <hints> -->
  <!--   <hint type="sourcelabel"> -->
  <!--    <x>20</x> -->
  <!--    <y>20</y> -->
  <!--   </hint> -->
  <!--   <hint type="destinationlabel"> -->
  <!--    <x>20</x> -->
  <!--    <y>20</y> -->
  <!--   </hint> -->
  <!--  </hints> -->
  <!-- </connection> -->
  <connection>
   <sender>messagelistInbox</sender>
   <signal>messageSelected(QString)</signal>
   <receiver>messageView</receiver>
   <slot>setContent</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
